x-pack/plugins/upgrade_assistant/__jest__/client_integration/overview/upgrade_system_indices/step_completition.test.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { act } from 'react-dom/test-utils';
+
+import { OverviewTestBed, setupOverviewPage } from '../overview.helpers';
+import { setupEnvironment, advanceTime } from '../../helpers';
+import { SYSTEM_INDICES_UPGRADE_POLL_INTERVAL_MS } from '../../../../common/constants';
+
+describe('Overview - Upgrade system indices - Step status', () => {
```

Comment:
This file name has a typo: `step_completition.test.ts` -> `step_completion.test.ts`